test/functional/ui/cursor_spec.lua

```diff
@@ -286,6 +286,21 @@ describe('ui/cursor', function()
       eq(173, named.normal.blinkon)
       eq(42, named.showmatch.cell_percentage)
     end)
+
+    -- If there is no setting for guicursor, it becomes the default setting.
+    meths.set_option('guicursor', 'n:ver35-blinkwait171-blinkoff172-blinkon173-Cursor/lCursor')
+    screen:expect(function()
+      for _,m in ipairs(screen._mode_info) do
+        if m.name ~= 'normal' then
```

Comment:
should we explicitly test the "normal" case also?

Reply:
I think the case where the setting exists in `guicursor` has been well tested in previous tests. Therefore, I don't think it's necessary.